### DISCUSSION TITLE
Welcome to Dynamic-Island-Sketchybar Discussions!

### DISCUSSION THREAD

**Discussion by crissNb** [crissNb/Dynamic-Island-Sketchybar#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas or new features that can come to dynamic island.
  * Engage with other community members.

If you encounter an issue to dynamic island config file, please use GitHub issues instead of GitHub discussions.

**Comment by ImW1ll** [comment]
Hi!
Great work! Really cool! Do you think on adding new features to this awesome tool in the future? I have been using for the last month and can't see how I would come back to the old ui.

**Reply by crissNb** [reply]
Thanks. Currently, I don't really have specific features (islands) planned to be added to the Dynamic Island. I did mention I'll potentially integrate on DND or bluetooth status, but I haven't got a time to get started with those yet. If you have any other features you'd like to see, please let me know.

**Reply by ImW1ll** [reply]
Hey man, do you know if there's a way to increased the number of frames on the animations? It feels slow and disconnected from the rest of the screen. In addition, the ui sometimes disappears when I change from brightness to volume too quickly. Is there a way to fix that? Thanks again for the whole thing its so cool.

**Reply by crissNb** [reply]
Thanks, the dynamic island normally shouldn't usually feel sluggish. 

> In addition, the ui sometimes disappears when I change from brightness to volume too quickly

Could you perhaps provide a screen recording to this?

It would be best if you could open up an issue addressing your problems along with screen recordings.

**Comment by FrendlyFireOnly** [comment]
This is awesome! 

I was just curious as to why it is always using the microphone while running?
